host-configs/darwin-clang.cmake

```diff
@@ -11,20 +11,26 @@ set(MPI_C_COMPILER "/usr/local/bin/mpicc" CACHE PATH "")
 set(MPI_CXX_COMPILER "/usr/local/bin/mpicxx" CACHE PATH "")
 set(MPIEXEC "/usr/local/bin/mpirun" CACHE PATH "")
 
-set(ENABLE_GTEST_DEATH_TESTS ON CACHE BOOL "" FORCE )
+set(ENABLE_GTEST_DEATH_TESTS ON CACHE BOOL "" FORCE)
 
 set(ENABLE_PAMELA ON CACHE BOOL "" FORCE)
 set(ENABLE_PVTPackage ON CACHE BOOL "" FORCE)
 set(ENABLE_GEOSX_PTP ON CACHE BOOL "" FORCE)
 
-set(CUDA_ENABLED      "OFF"       CACHE PATH "" FORCE)
-set(ENABLE_OPENMP     "OFF"        CACHE PATH "" FORCE)
+set(CUDA_ENABLED "OFF" CACHE PATH "" FORCE)
+set(ENABLE_OPENMP "OFF" CACHE PATH "" FORCE)
 
-option( ENABLE_CALIPER "Enables CALIPER" OFF )
+option(ENABLE_CALIPER "Enables CALIPER" OFF)
 
-set( BLAS_LIBRARIES /usr/lib/libblas.dylib CACHE PATH "" FORCE )
-set( LAPACK_LIBRARIES /usr/lib/liblapack.dylib CACHE PATH "" FORCE )
+set(BLAS_LIBRARIES /usr/lib/libblas.dylib CACHE PATH "" FORCE)
+set(LAPACK_LIBRARIES /usr/lib/liblapack.dylib CACHE PATH "" FORCE )
+
+set(ENABLE_DOXYGEN OFF CACHE BOOL "" FORCE)
 
-set( ENABLE_DOXYGEN OFF CACHE BOOL "" FORCE )
 #set( DOXYGEN_EXECUTABLE /usr/local/bin/doxygen CACHE PATH "" FORCE )
-#set( SPHINX_EXECUTABLE /usr/local/bin/sphinx-build CACHE PATH "" FORCE )
\ No newline at end of file
+#set( SPHINX_EXECUTABLE /usr/local/bin/sphinx-build CACHE PATH "" FORCE )
+
+set(ENABLE_GEOSX_PTP ON CACHE BOOL "" FORCE)
+
+set(GEOSX_TPL_DIR "/usr/local/GEOSX/GEOSX_TPL" CACHE PATH "" FORCE)
```

Comment:
This will break the build on my laptop won't it?

Reply:
Yeah it will. Eventually with Spack we will have a separate host-config for each platform. We can probably do it in such a way (untar the TPLs inside the repo for example) that you could use the same host config.

Reply:
Actually I take that back, in theory you could make it so that you could use the travis OSX host-config, since you'll be building the TPLs with spack anyways you should just use the host-config that spack spits out. We should only put host-configs in the repo for situations where we don't expect users to build the TPLs (LC and Docker).